public/index.html

```diff
@@ -1,121 +1,247 @@
+
+  <div class="card">
+
+    <div class="survey">
+      <h1>Which of these technologies do you use?</h1>
+      <div class="option">
+        <div class="checkbox">
+          <svg xml:space="preserve" focusable="false" version="1.1" viewBox="0 0 24 24">
+            <path class="checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none"></path>
+          </svg>
+        </div>
+         Skaffold
+      </div>
+      <div class="option">
+        <div class="checkbox">
+          <svg xml:space="preserve" focusable="false" version="1.1" viewBox="0 0 24 24">
+            <path class="checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none"></path>
+          </svg>
+        </div>
+        Jib
+      </div>
+      <div class="option">
+        <div class="checkbox">
+          <svg xml:space="preserve" focusable="false" version="1.1" viewBox="0 0 24 24">
+            <path class="checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none"></path>
+          </svg>
+        </div>
+        Knative
+      </div>
+      <div class="option">
+        <div class="checkbox">
+          <svg xml:space="preserve" focusable="false" version="1.1" viewBox="0 0 24 24">
+            <path class="checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none"></path>
+          </svg>
+        </div>
+        App Engine
+      </div>
+      <div class="option nota">
+        <div class="checkbox">
+          <svg xml:space="preserve" focusable="false" version="1.1" viewBox="0 0 24 24">
+            <path class="checkmark-path" d="M4.1,12.7 9,17.6 20.3,6.3" fill="none"></path>
+          </svg>
+        </div>
+        None of the above
+      </div>
+      <div class="submit-btn">Submit</div>
+    </div>
+
+    <h2 class="thanks">Thanks for voting! <br>Checkout the <a href="/results">results</a>.</h2>
+
+  </div>
+
 
-    document.querySelector('#submit').classList.add('loading');
-
-    fetch(url, {
-      method: 'POST',
-      body: JSON.stringify(votes),
-      headers: {
-        'Content-Type': 'application/json; charset=utf-8'
-      },
-    }).then((response) => {
-      document.querySelector('#submit').classList.remove('loading');
-      console.log(response);
-      if (response.ok) {
-        document.querySelector('#options').style.display = 'none';
-        document.querySelector('#middle').style.display = 'table';
+  <script>
+    const optionData = ['Skaffold', 'Jib', 'Knative', 'App Engine', 'None of the above'];
```

Comment:
Hey Ric, in the index page, it looks like the names here repeated twice.
Is it possible to consolidate them into one?

I am asking this because, as part of this demo, we are trying to change the questionnaire and editing them at multiple places can be overwhelming.

Reply:
If possible, can we use the array so I can copy and paste my changes to the results page as well?